app/styles/variables.css

```diff
@@ -19,4 +19,6 @@
   --color-green: #49a82e;
 
   --color-light-navyblue: #1d128340;
+
+  --color-cultured: #f6f6f6;
```

Comment:
I think we can use already declared variable for this color

Reply:
https://github.com/Real-Dev-Squad/website-www/pull/446#discussion_r1181047022